include/boost/gil/extension/dynamic_image/any_image.hpp

```diff
@@ -100,6 +100,9 @@ class any_image : public variant2::variant<Images...>
 
     template <typename Image>
     explicit any_image(Image const& img) : parent_t(img) {}
+    
+    template <typename Image>
+    any_image(Image&& img) : parent_t(std::move(img)) {}
```

Comment:
By the way, I forgot to ask about this around 601790f2419c90b836dc3cfa7bb824347b1a2369, please add yourself copyright to the copyright notice

Reply:
Done